about.html

```diff
@@ -27,6 +27,8 @@ <h2 class="Who-we-Are">Who we Are</h2>
         that allows you to review the latest movies you've watched. If you have ever felt the need for reviewing movies and TV series, 
         here your search comes to an end. As an up and coming website, we're constantly looking forward to working with our userbase and
         helping our users feel more welcomed.
+
+        This is a project developed by <a href="https://github.com/AjaybirRandhawa" style="color:#bb86fc">Ajaybir Randhawa</a>, <a href="https://raynorelgie.com/" style="color:#bb86fc">Raynor Elgie</a>, <a href="https://www.linkedin.com/in/finn-huynh/" style="color:#bb86fc">Finn Huynh</a>, and <a href="https://www.linkedin.com/in/belleville-max/" style="color:#bb86fc">Maximillian Belleville.</a> 
```

Comment:
I used whatever links I both had on hand and I felt were appropriate, you can modify your own after the merge.